apps/e2e/ag-grid-storybook/src/app/data-manager/data-manager.component.scss

```diff
@@ -30,6 +30,10 @@ app-data-manager.use-normal-dom-layout {
     height: 100%;
     width: 100%;
   }
+  .sky-data-manager-content {
+    height: 100%;
+    width: 100%;
+  }
```

Comment:
This change was added to make sure that the screenshots for ag grid inside a data manager passed. But it gave me pause that this could cause breaking changes for consumers visually.

For ag grid to display properly in storybook every parent in the stack above `.ag-body` needed height 100%. This file in our e2e folder does that for our visual tests. But my PR adds a new div that did not have that, so this change was needed. Idk if that is just a storybook quirk though bc serving the app we use for e2e tests normally using `nx serve` did not have an issue